source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -0,0 +1,310 @@
+===================
+Search Geospatially
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+On this page you will learn the how to search **geospatial data** in the
+MongoDB Java Driver.
+
+Geospatial data is data that represents a geographical location on
+the surface of the Earth. Examples of geospatial data include:
+
+- Locations of movie theaters
+- Borders of countries
+- Routes of bicycle rides
+- Dog exercise areas in New York City
+
+You should read this guide if you would like to:
+
+- Understand the different geospatial data formats in MongoDB.
+- Build geospatial queries.
+
+Coordinates on an Earth-like Sphere
```

Comment:
Here and throughout, I'd remove "earth-like". I don't believe MongoDB supports defining a custom center point, radius, equator, and meridian?

Reply:
I think this is correct, I can't find anything suggesting you can define your own sphere. This might be worth making a ticket for as the term "Earth-Like" seems to be used throughout most of our documentation.

https://docs.mongodb.com/manual/core/2dsphere/
https://docs.mongodb.com/manual/geospatial-queries/
https://docs.mongodb.com/drivers/node/current/fundamentals/crud/read-operations/geo/

Might be worth making a ticket to change these from "an earth-like sphere" to just "earth". 

Removed "earth-like sphere" from page: